smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/CommandGenerator.java

```diff
@@ -180,7 +180,10 @@ private String getCommandExample(String serviceName, String configName, String c
                                 model.getShape(operation.getInputShape()).get(), model))
                 + String.format("const command = new %s(input);%n", commandName)
                 + "const response = await client.send(command);\n"
-                + "```\n"
+                + String.format("/**%n%s%n",
```

Comment:
This generates a comment block opening indicator `/**`, right? Does it need to be escaped, since it's an example block inside what is already a comment block?

Reply:
The opening indicator doesn't need to be escaped, but the closing indicator would, which is why I left it out.  TSDoc's extractor reads this implementation correctly as the example is closed by "```"